docs/document/package.json

```diff
@@ -1,28 +1,45 @@
 {
   "name": "document",
+  "version": "1.0.0",
+  "private": true,
   "scripts": {
-    "lint-md": "remark .",
-    "es-lint": "node_modules/.bin/eslint website --ext .js"
+    "docusaurus": "docusaurus",
```

Comment:
For creating version doc, how about adding script alias for it?
"npm run docusaurus docs:version 1.12.0"

Reply:
How can I operate in case of below?
1. update EN page
2. create new v1.12.0 with `npm run docusaurus docs:version 1.12.0`
3. update JA page
4. I coudln't re-create v1.12.0 with the above command and got an error it's already exist.

Reply:
You have to delete that version and create one again
https://docusaurus.io/docs/versioning#deleting-an-existing-version
You can view more information at dev#10 No.5

Reply:
> In my opinion, we should use this command

What is the meaning of this?

> You have to delete that version and create one again

Got it.

Reply:
>What is the meaning of this?
I mean we can use ``npm run docusaurus docs:version 1.12.0`` when we create a new version.
I think we don't need adding script alias for it

Reply:
> I mean we can use npm run docusaurus docs:version 1.12.0 when we create a new version.
> I think we don't need to add script alias for it.

Got it. I agree with it.

> I have a solution for resolve this problem
> When we work on SSR task, we only update on 3 directories below:
> If it is Ok, when we release the new version (release task), we will be following this step below:
> type: npm run docusaurus docs:version 1.12.0

I see.
If doing so, developer can check the new updated version under the previous version with localhost, right?

Reply:
> If doing so, developer can check the new updated version under the previous version with localhost, right?
If we have many version document : 1.0.0 ; 1.0.1 ....1.11.0 and now we need to create 1.12.0
We can see the newest version (1.12.0) at this link
- localhost:3000/docs/**next**/components/desktop/..
- localhost:3000/**ja**/docs/**next**/components/desktop/..
If you want to check old version, you can access the link which removed **next** character on URL

> If so, we should consult with PG members about the operation.
Yes, I have update operation at here: https://sharedoc.atlassian.net/l/cp/t9peBj9V

Reply:
I see! We can check the latest version under next/.
Thanks for updating the guide. I think it's good idea and let's consult with members.